app/models/loan.rb

```diff
@@ -118,23 +121,23 @@ def self.status_active_id
         [8, 'Relacion Activo']
       ])
 
-  LOAN_TYPE_OPTIONS = OptionSet.new(
-      [ [1, "Liquidity line of credit"],
-        [2, "Investment line of credit"],
-        [3, "Investment Loans"],
-        [4, "Evolving loan"],
-        [5, "Single Liquidity line of credit"],
-        [6, "Working Capital Investment Loan"],
-        [7, "Secured Asset Investment Loan"]
-      ])
-
-  PROJECT_TYPE_OPTIONS = OptionSet.new(
+  # LOAN_TYPE_OPTIONS = TransientOptionSet.new(
+  #     [ [1, "Liquidity line of credit"],
+  #       [2, "Investment line of credit"],
+  #       [3, "Investment Loans"],
+  #       [4, "Evolving loan"],
+  #       [5, "Single Liquidity line of credit"],
+  #       [6, "Working Capital Investment Loan"],
+  #       [7, "Secured Asset Investment Loan"]
+  #     ])
+
+  PROJECT_TYPE_OPTIONS = TransientOptionSet.new(
       [ [1, 'Conversion'],
         [2, 'Expansion'],
         [3, 'Start-up'],
       ])
 
-  PUBLIC_LEVEL_OPTIONS = OptionSet.new(
+  PUBLIC_LEVEL_OPTIONS = TransientOptionSet.new(
       [ [1, 'Featured'],
         [2, 'Hidden'],
       ])
```

Comment:
Are these TransientOptionSets only for migration? If so, can they live in Legacy::Loan?